### ISSUE TITLE
Research table freezes the game on multiplayer server

### ISSUE THREAD

**Issue by Arribaox** [Azanor/thaumcraft-beta#1593]
every time a player opens or theorizes in the research table this freezes the game of all for a few seconds, also when you choose a theory you must close and open the interface to continue causing it to freeze again

**Comment by Celmas** [comment]
Same problem. Have u fixed it?

**Comment by Arribaox** [comment]
No, I did not, but once my team and I played for 4 hours, the problem disappeared for a while, although when restarting the server it reappeared

**Comment by 256leon** [comment]
already checked for to low RAM? (MC and Java)
press F3 to see

**Comment by Arribaox** [comment]
5Gb to server and 4Gb to MC
I have a total of 12Gb

**Comment by 256leon** [comment]
how many additional mods do you have? could you post a log?
